packages/renderer-preact/package.json

```diff
@@ -3,7 +3,8 @@
   "browser": "dist/es/index.js",
   "description": "A SkateJS renderer for Preact.",
   "devDependencies": {
-    "preact": "8.2.7"
+    "preact": "8.2.7",
+    "skatejs": "^5.1.2"
```

Comment:
Added this since the tests are now pulling in `skatejs` in https://github.com/skatejs/skatejs/commit/c8b0006bf70eab9790f2e7169c68f5468b800618#diff-61ecfdc8a7d04b1bf1fbefe49953d7feR4